plugin/src/main/kotlin/org/gradle/github/dependency/GitHubDependencySubmissionPlugin.kt

```diff
@@ -0,0 +1,24 @@
+package org.gradle.github.dependency
+
+import org.gradle.api.Plugin
+import org.gradle.api.invocation.Gradle
+import org.gradle.api.logging.Logging
+import org.gradle.github.dependency.extractor.ForceDependencyResolutionPlugin
+import org.gradle.github.dependency.extractor.GitHubDependencyExtractorPlugin
+
+@Suppress("unused")
+class GitHubDependencySubmissionPlugin : Plugin<Gradle> {
+
+    private val LOGGER = Logging.getLogger(GitHubDependencySubmissionPlugin::class.java)
```

Comment:
I recommend moving this logger to a companion object exactly as is done in `GitHubDependencyExtractorPlugin`.

Reply:
Done df3dd03